js/lib/entryPoints/feedback.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import * as ReactDOM from "react-dom/client";
+import Highlighter from "web-highlighter";
+
+import SelectionRenderer from "../components/feedback/selection";
+
+export const HIGHLIGHT_STORAGE_KEY = "mdbook-quiz:highlights";
+
+let initFeedback = () => {
+  let highlighter = new Highlighter();
```

Comment:
Is there a reason to make this a prop to `SelectionRenderer` as opposed to local state within `SelectionRenderer`?

Reply:
Factoring out `FeedbackRenderer` now makes this necessary.